Source/Blazorise.Bootstrap5/Styles/components/_table.scss

```diff
@@ -99,4 +99,41 @@ tr {
 
 td:focus {
     outline: -webkit-focus-ring-color auto 1px;
+}
+
+@media only screen and (max-width: 760px), (min-device-width: 768px) and (max-device-width: 1024px) {
+
+    .b-table-mobile {
```

Comment:
The classnames should follow the CSS provider naming, as close as it can. For BS it should be `table-mobile`.